files/es/learn/javascript/first_steps/a_first_splash/index.html

```diff
@@ -178,7 +178,7 @@ <h6 id="Top_hidden_code">Top hidden code</h6>
 
 <ol>
  <li>Generar un número aleatorio entre 1 y 100.</li>
- <li>Registrar el número del intento en el que el jugador se encuentre. Empezando en 1.</li>
+ <li>Registrar el número de intentos en el que el jugador se encuentre. Empezando en 1.</li>
```

Comment:
En este caso a mención a una posición en la iteración de intentos y no un acumulativo de los mismos.

```suggestion
 <li>Registrar el número del intento en el que el jugador se encuentre. Empezando en 1.</li>
```